op-node/node/node.go

```diff
@@ -229,27 +261,26 @@ func (n *OpNode) initRuntimeConfig(ctx context.Context, cfg *Config) error {
 				case errNodeHalt, nil:
 					n.log.Debug("reloaded runtime config", "l1_head", l1Head)
 					if err == errNodeHalt {
```

Comment:
you use `errors.Is` above to check for `errNodeHalt`, maybe a good opportunity to fix it here too.